test/NinjaTools.FluentMockServer.Tests/TestHelpers/Data/TheoryData/EqualityOperatorTestData.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Reflection;
+using JetBrains.Annotations;
+using NinjaTools.FluentMockServer.FluentAPI;
+using NinjaTools.FluentMockServer.FluentAPI.Builders;
+using NinjaTools.FluentMockServer.FluentAPI.Builders.ValueObjects;
+using NinjaTools.FluentMockServer.Models;
+using NinjaTools.FluentMockServer.Models.HttpEntities;
+using NinjaTools.FluentMockServer.Models.ValueTypes;
+using NinjaTools.FluentMockServer.Tests.TestHelpers.Data.Random;
+using Xunit;
+
+namespace NinjaTools.FluentMockServer.Tests.TestHelpers.Data.TheoryData
+{
+    public class EqualityOperatorTestData<T> : TheoryData<T, T, bool> where T : class
+    {
+        private static Body CreateBody([NotNull] Action<IFluentBodyBuilder> factory)
+        {
+            var type = typeof(FluentBodyBuilder);
+            var builder = new FluentBodyBuilder();
+            factory(builder);
+            
+            var bodyProperty = type.GetProperty("Body", BindingFlags.NonPublic | BindingFlags.Instance);
+            var body = bodyProperty?.GetValue(builder) as Body;
+
+            return body;
+        }
+        
+        public EqualityOperatorTestData()
+        {
+            var dataType = typeof(T);
+            if (typeof(Body) == dataType)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4722826)